pkg/config/analysis/msg/messages.yaml

```diff
@@ -548,9 +548,33 @@ messages:
         type: string
       - name: pod
         type: string
-        
+
   - name: "ExternalNameServiceTypeInvalidPortName"
     code: IST0150
     level: Warning
     description: "Proxy may prevent tcp named ports and unmatched traffic for ports serving TCP protocol from being forwarded correctly for ExternalName services."
-    template: "Port name for ExternalName service is invalid. Proxy may prevent tcp named ports and unmatched traffic for ports serving TCP protocol from being forwarded correctly"
\ No newline at end of file
+    template: "Port name for ExternalName service is invalid. Proxy may prevent tcp named ports and unmatched traffic for ports serving TCP protocol from being forwarded correctly"
+
+  - name: "DestinationRuleHostShouldNotBeShortName"
```

Comment:
I still don't understand why we are adding this. This seems like we are making an *extremely* common pattern an 'error'. 

The original issue was about a bug in an existing analyzer. I don't know why more analyzers are being added here?

Reply:
In fact, this is not adding new analyzer, it's adding new warning message for the analyzer.  And it's okay to remove this unnecessary message.